### ISSUE TITLE
Kohei needs to inspect kohei/needs_inspection branch

### ISSUE THREAD

**Issue by kbenoit** [kbenoit/quanteda#284]
@koheiw, here is where I left the merge and cleanup of your working branches.

## branch `kohei/needs_inspection`

`dev_grams` and `dev_joinTokens` are merged into this branch, so all the work that has been done on the token hashing is in there. In fact there is probably also **much more** in there than we actually need, since I think there are now multiple versions of the C++ code for things like ngrams and joinTokens.

I'd like you do go through the `/src`, `joinTokens.R` and `ngrams.R` carefully, removing anything that is old, but NOT BREAKING anything old, including the methods for the now older `tokenizedTexts` class object class.

Please do one thing at a time, run CHECK, and make one commit at a time **to this branch only**. Once you are satisfied that it's tidy and working as expected, you can issue a PR, but only if it passes CHECK.

## branch `kohei/benchmark_hashing`

I renamed `benchmark_hashing` to `kohei/benchmark_hashing`. When you have sorted out the above, and it is merged into master, then you can rescue the benchmarks from that branch. But please, **only after the above branch is resolved, merged into master, and deleted**.

## branch `archived/dev_selectFeatures_mt`

On GitHub you cannot "archive" a branch, so I renamed `dev_selectFeatures_mt` to `archived/dev_selectFeatures_mt`. Once everything else is working, we can return to the parallelised code issue.

**Comment by koheiw** [comment]
I inspected the branch and found it looks fine overall. I also removed files and functions for multi-thread select features, which no one is using at the moment. It also passed the CHECK, so I pushed it. There are many C++ files simply because we are keeping functions for tokenizedTexts objects. I am more than happy to delete all of those when time comes.
